src/routing_message_filter.rs

```diff
@@ -70,9 +72,15 @@ impl RoutingMessageFilter {
 
     // Filter outgoing `RoutingMessage`. Return whether this specific message has been seen recently
     // (and thus should not be sent, due to deduplication).
+    //
+    // Return `false` if serialisation of the message fails - that can be handled elsewhere.
```

Comment:
Added a trace message (initially added an `error!` 'cause I figured it was unreachable, but even if it is reachable, we probably don't want to spam people with an error here).